Jenkinsfile

```diff
@@ -242,8 +242,8 @@ pipeline {
                                 unstash 'MathSetup'
                                 sh "echo CXX=${env.CXX} -Werror > make/local"
                                 sh "echo STAN_OPENCL=true>> make/local"
-                                sh "echo OPENCL_PLATFORM_ID=0>> make/local"
-                                sh "echo OPENCL_DEVICE_ID=${OPENCL_DEVICE_ID}>> make/local"
+                                sh "echo OPENCL_PLATFORM_ID=${env.OPENCL_PLATFORM_ID}>> make/local"
+                                sh "echo OPENCL_DEVICE_ID=${env.OPENCL_DEVICE_ID}>> make/local"
```

Comment:
Mind adding the same for the Windows machine as well?

Reply:
@serban-nicusor-toptal also we can change the label to "gelman-group-win2 || linux-gpu" and then add linux-gpu to the AWS instance and the gg-linux?

Reply:
Sure! This is actually a great idea to simplify matters.